building/glass/branches-schema.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: "http://json-schema.org/draft-07/schema#"
+title: BranchesConfig
+type: object
+properties:
+  branches:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+          pattern: "[a-z0-9._-]+/[a-z0-9_-]+"
```

Comment:
Why aren't we supporting branches with multiple slashes, or dots after the last slash? (I don't remember exactly, but I think we implicitly supported those previously.)

Reply:
Ignore this nitpick.

Reply:
Good catch with the period, though. I had royally messed up the regexes in this file in more ways than one.